example/App_new.res

```diff
@@ -0,0 +1,118 @@
+// open Components
+
+// module Values = {
+//   @lenses
+//   type t = {
+//     age: string,
+//     name: string,
+//     password: string,
+//     passwordConfirm: string,
+//     email: string,
+//   }
+
+//   let init = {
+//     age: "",
+//     name: "",
+//     password: "",
+//     passwordConfirm: "",
+//     email: "",
+//   }
+// }
+
+// // It's pretty easy to have a generic form maker
+// // for when a the same value set and errors are used in different context
+// let makeForm = Formidable.make(~values=module(Values), ~error=module(I18n.Error))
+
+// // In this example, we'll use the hook contained in the module so we need to define it at the top level
+// // In many cases, it's not needed, especially thanks to the Consumer component
+// module Form = (
+//   val makeForm(~onSubmit=_values => (), ~onSubmitError=(_values, _errors) => (), ())
```

Comment:
I tried to covert this file as well but currently I got struggle in this line, and cannot resolve it for now